docs/contributing/index.rst

```diff
@@ -3,59 +3,32 @@
 Contributing to Weblate
```

Comment:
```suggestion
Non-code contributions
```
In the interest of keeping the sidemenu short, and keeping code and code approaches in that section.

Reply:
This page was a general contributing landing page. I don't think removing the code part is a good idea.

Also, this would change the label of all references to this section and that would make it harder to understand - "See also: Contributing to Weblate" is clear, while "See also: Non-code contributions" is not.

Reply:
What on earth is a "landing page" for contributions, if not something that wastes attention, short of actually conveying any useful info. It is a suggestion, because it may break assumptions.

There is no different way of contributing translations for Weblate, translating other projects also helps the general esteem of Weblate, but also the shared TM (for what is actually works between Weblate and other projects on Hosted).

As such, a "Contributions" category is a mistake. This all belongs in the general user docs, with Hosted Weblate/Weblate used only as an example. That way other projects can integrate it in their guidelines. Bonus points for placeholders instead of hardcoded Hosted Weblate/Weblate, but alas.

Reply:
@comradekingu

If two people from the core Weblate team are telling you something, please consider it has some reason instead of telling everyone it’s a mistake or "what on the Earth...".

These follow-up PRs are not counters, please don’t take it personally. It’s an evolution. If creating a follow-up PR makes more sense than correcting something hardly mergeable, it’s not a personal attack. It’s not ideal, but it’s the best way, sometimes. The original author can still collaborate and give it their best.

Reply:
The distinction of who is on the core team is meaningless to me. I respect the work of people that do good work.
Two people from the core Weblate team is telling me what exactly? Second time you pulled rank.
What is the reason there? I am telling the world at large I don't think changing to "non-code contribution" is a change without its issues. Hence "suggested change".
Also, I can get links to actually useful resources back in to facilitate phone-readers. I OTOH read it on a 3860 px vertical screen, and the sidemenu has enough entries to where it almost scrolls…

Speaking of, your guide is _a lot_ longer, that is _a lot_ worse for mobile-readers. It is worse for blind people, it is worse for people looking to find and use info. There is no overview to be had.

You are hung up in language, rather than arguing the point, which is "short of" part.
There are reasons here. Additionally, you have replicated info found elsewhere, but reworded it. There is nothing personal there, nor am I taking it that way. It is not an evolution. You going your own way is the opposite.
That you work in corrections I make is not an evolution of the origin. It is a flat refusal to entertain a discussion you invited, for what you said was a basis to work on.

Sometimes shutting down a PR is the best option, yes. When you yolo in a page that is your own, it would have been.
Evidently, your version is full of mistakes. And still is.
The original author is you because you decided to be with commit access, not because I respect your work.
It can sit there and be laughable. Nothing better than seeing my own language tarnished, in a further blight to the resource I worked on. What's not to like? "Landing page" in the documentation awful, newest blogpost basic spelling errors.
Why are you in charge of that?

How do you make it past whatever workflow you have with that? Why are you beyond the actual critique you don't even care to offer others?

I can possibly "check and comment", but I didn't even know it existed.

Put, it, on, review, you are the public image of the one libre translation platform we have.
There is a "reviewers" team. Don't know if it is called "documentation", and it would be a welcomed change if either started to be part of what that group actually does.

Your edit also says donations are used to host other projects. That is not what it says on the actual donation page.
Nor is it actually something that belongs in "Contributing to Weblate". The psychology is sub-par with that angle.
Not my call to decide how funds are used, or what to merge, but I used my time to hone the entire string-base of the website.

"While this isn’t mergeable", means _what_ exactly, seeing as you wrote it as you closed this PR?

Reply:
> Also, I can get links to actually useful resources back in to facilitate phone-readers.

All I said is that it makes navigating the documentation for some readers worse. It can be certainly improved (anything can be improved), but it cannot be merged as it is.

This PR exactly shows reasons why we've added a guideline on contributing to the documentation, which you have removed. This PR contains parts which could be more or less merged right now (and @orangesunny extracted those to https://github.com/WeblateOrg/weblate/pull/5697), parts which need additional work and parts which are not acceptable. Doing smaller pull requests makes it easier to review, merge and improve the documentation in smaller steps.

Reply:
@comradekingu
I am sorry to see that you switched to a way in which you are picking anything to call it bad, worse, a mistake, using strong language. This is something we can hardly work with. I will be happy if you switch back to the constructive way I know you can navigate pretty well.

Please drop me a message on Wire, or in an e-mail, let’s talk and clear the air so we can understand each other better and continue to cooperate productively.

To summarize this issue from my point of view:
1. I wrote the https://github.com/WeblateOrg/weblate/pull/5658
   - to include what Weblate users and customers were asking for.
   - to prevent the issues we bumped into while working on *Access* article from happening again.
   - merge it together with @nijel. Once it’s released, everybody can read it, respect it, and suggest a change if needed.
2. You read it and suggested a change here (https://github.com/WeblateOrg/weblate/pull/5688), changing the structure, purpose, and deleting the whole new part.
3. As it is a pull "request", we checked it with @nijel and realized it’s not mergeable. So ⬇️ 
4. I tried to use all of your good parts from here in https://github.com/WeblateOrg/weblate/pull/5697 and ask you to comment there so we can make it good together. And you start down-talking it and saying you are not able to work with that.

Again, I am sorry to read you are not happy with it and I am sure we can find a way to cooperate. Let’s talk.

I believe that we will write and link the materials that will help us and newcomers to orient in the docs’ contribution, show how and which syntaxes to use, and make docs writing easier and more effective for everyone.